README.md

```diff
@@ -13,22 +13,92 @@ Ansible role for 7.x/6.x Elasticsearch.  Currently this works on Debian and RedH
+es_data_dirs:
+  - /var/lib/elasticsearch/{{ node_name }}-{{ instance_name }}
+es_log_dir: /var/log/elasticsearch/{{ node_name }}-{{ instance_name }}
+es_pid_dir: /var/run/elasticsearch/{{ node_name }}-{{ instance_name }}
+```
+
+2. Deploy ansible-role. **Even if these variables are overrided, Elasticsearch config file and default option file will change, which imply an Elasticsearch restart.**
+
+3. After ansible-role new deployment, you can do some cleanup of old Init file and Default file.
+
+Example:
+```bash
+$ ansible-playbook -e '{"es_conf_dir":"/etc/elasticsearch/node1","es_data_dirs":["/var/lib/elasticsearch/localhost-node1"],"es_log_dir":"/var/log/elasticsearch/localhost-node1","es_pid_dir":"/var/run/elasticsearch/localhost-node1"}' playbook.yml
+...
+TASK [elasticsearch : Create Directories] **********************************************************************************************************************************************************************************************************************
+ok: [localhost] => (item=/var/run/elasticsearch/localhost-node1)
+ok: [localhost] => (item=/var/log/elasticsearch/localhost-node1)
+ok: [localhost] => (item=/etc/elasticsearch/node1)
+ok: [localhost] => (item=/var/lib/elasticsearch/localhost-node1)
+
+TASK [elasticsearch : Copy Configuration File] *****************************************************************************************************************************************************************************************************************
+changed: [localhost]
+
+TASK [elasticsearch : Copy Default File] ***********************************************************************************************************************************************************************************************************************
+changed: [localhost]
+...
+PLAY RECAP *****************************************************************************************************************************************************************************************************************************************************
+localhost                  : ok=32   changed=3    unreachable=0    failed=0
+
+$ find /etc -name 'node1_elasticsearch*'
+/etc/default/node1_elasticsearch
+/etc/systemd/system/multi-user.target.wants/node1_elasticsearch.service
+$ rm /etc/default/node1_elasticsearch /etc/systemd/system/multi-user.target.wants/node1_elasticsearch.service
+```
+
+## Dependency
 This role uses the json_query filter which [requires jmespath](https://github.com/ansible/ansible/issues/24319) on the local machine.
 
 ## Usage
 
 Create your Ansible playbook with your own tasks, and include the role elasticsearch. You will have to have this repository accessible within the context of playbook.
 
 ```sh
-ansible-galaxy install elastic.elasticsearch,7.0.1
+ansible-galaxy install git+https://github.com/elastic/ansible-elasticsearch.git,7f5be969e07173c5697432141e909b6ced5a2e94
```

Comment:
This commit is fine. There is only https://github.com/elastic/ansible-elasticsearch/commit/25bd09f6835b476b6a078676a7d614489a6739c5 after this one with no change in ansible code. Morever, this will be changed in next PR with changelog and git tag release.